docker-compose.yml

```diff
@@ -32,6 +33,8 @@ services:
     ports:
       - 8001:8000
     restart: on-failure
+    extra_hosts:
+      - host.docker.internal:host-gateway
```

Comment:
@toch wouldn't it make sense to use the `openhexa` network that we recommend to create in the `README.md` of [`openhexa-app`](https://github.com/BLSQ/openhexa-app)?

Or could we drop this network altogether?